find_target_name_from_header.py

```diff
@@ -212,8 +193,6 @@ def query_simbad(date,coords,name=None,limit_G_mag=15,metadata=None,verbose=True
                   validSearch['MAIN_ID'][i_min],validSearch['FLUX_G'][i_min],min_sep))
         simbad_dico = populate_simbad_dico(validSearch,i_min,simbad_dico)
         simbad_dico = add_separation_between_pointing_current_position(coords,simbad_dico)
-        if verbose:
```

Comment:
Could I ask you to keep this description if verbose is True ? I find it easier to debug with that verbose

Reply:
Sure! Fixed